### DISCUSSION TITLE
Build multiple bins with single command

### DISCUSSION THREAD

**Discussion by sam-rusty** [cargo-lambda/cargo-lambda#675] [Ideas]
It would be nice to have a command that build all bins with single command, let's say i have 50 bins/functions in my app, i have to run build command run one by one for each, if i could have a single command (or a flag) that can build all bins at once, would save a lot of time.

**Comment by calavera** [comment]
`cargo lambda build` without any flags should build all binaries in a workspace or package with multiple binaries. It behaves the same way as `cargo build` when you're in a workspace or you have a package with multiple binaries. Did you try that?

**Reply by sam-rusty** [reply]
I might have overlooked that, yeah, that works. thanks <3
